.github/workflows/setup_git.ps1

```diff
@@ -9,8 +9,3 @@ if ($(git branch --show-current) -ne "develop")
 {
     git branch develop origin/develop
 }
-
```

Comment:
https://github.com/spack/spack/pull/29565 does this plus removing one extra reference - mind if we go with that?